ecs-cli/modules/cli/servicediscovery/servicediscovery_app.go

```diff
@@ -14,97 +14,46 @@
 package servicediscovery
 
 import (
-	"fmt"
-
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/context"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/cloudformation"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+	utils "github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/compose"
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/ecs"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
-const (
-	privateDNSNamespaceStackNameFormat     = "amazon-ecs-cli-setup-%s-%s-private-dns-namespace"
-	serviceDiscoveryServiceStackNameFormat = "amazon-ecs-cli-setup-%s-%s-service-discovery-service"
-)
-
-// CloudFormation template parameters
-const (
-	parameterKeyNamespaceDescription = "NamespaceDescription"
-	parameterKeyVPCID                = "VPCID"
-	parameterKeyNamespaceName        = "NamespaceName"
-)
-
-const (
-	parameterKeySDSDescription                          = "SDSDescription"
-	parameterKeySDSName                                 = "SDSName"
-	parameterKeyNamespaceID                             = "NamespaceID"
-	parameterKeyDNSType                                 = "DNSType"
-	parameterKeyDNSTTL                                  = "DNSTTL"
-	parameterKeyHealthCheckCustomConfigFailureThreshold = "FailureThreshold"
-)
-
-const (
-	CFNTemplateOutputPrivateNamespaceID = "PrivateDNSNamespaceID"
-	CFNTemplateOutputSDSARN             = "ServiceDiscoveryServiceARN"
-)
-
-const (
-	dnsRecordTypeA   = "A"
-	dnsRecordTypeSRV = "SRV"
-)
-
-var requiredParamsSDS = []string{parameterKeyNamespaceID, parameterKeySDSName, parameterKeyDNSType}
-var requiredParamsNamespace = []string{parameterKeyVPCID, parameterKeyNamespaceName}
-
 // Create creates resources for service discovery and returns the ID of the Service Discovery Service
```

Comment:
Can we be more explicit about which resources are being created here in the description?